docs/using.rst

```diff
@@ -378,10 +378,15 @@ then restart it after the plugin is finished. Example::
 
   certbot renew --pre-hook "service nginx stop" --post-hook "service nginx start"
 
-The hooks will only be
-run if a certificate is due for renewal, so you can run this command
-frequently without unnecessarily stopping your webserver. More
-information about renewal hooks can be found by running
+If a hook exits with a non-zero exit code, the error will be printed
+to ``stderr`` but renewal will be attempted anyway. A failing hook
+doesn't directly cause Certbot to exit with a non-zero exit code, but
+if it causes renewal to fail it will have the same effect. Hooks will
```

Comment:
@zjs so let's say you have some certs that were retrieved using the standalone authenticator. If you have `systemctl stop apache2; false` as your pre-renewal hook, `false` will cause the hook's exit code to be `1` but the renewal will still succeed and Certbot's exit code will be `0`.

However, let's say you typo'd the first command so your pre-renewal hook was e.g. `systemctl stp apache2`. In that case, renewal would fail (because Apache wouldn't stop and the authenticator wouldn't be able to bind to the relevant port) and Certbot's exit code would be `1`.